src/pages/EMPReplicantTargets.page

```diff
@@ -0,0 +1,18 @@
+<apex:page standardController="EMP_Organization_Member__c" extensions="EMPReplicantProfile" sidebar="false" language="es">
+<apex:form >
+    <apex:pageBlock >
+      <apex:pageBlockSection title="Metas">
+        <apex:pageBlockTable value="{!targetDiagnostic}" var="fTD">
+            <apex:column headerValue="Fecha Visita">
+            <apex:outputLink target="_blank" value="/{!fTD.Id}">{!fTD.Visit_Date__c}</apex:outputLink>
```

Comment:
Yes, I mean showing the date in a different format. Thanks for the link, I did the modification.